src/plugins/intel_gpu/src/graph/reshape.cpp

```diff
@@ -97,14 +97,13 @@ padding propagate_padding(const layout& in_layout, const ov::PartialShape& out_s
         }
     }
 
-    auto convert_pad = [](const std::vector<int32_t> pad) {
-        return tensor(format::get_default_format(pad.size()), pad, 0);
-    };
-
-    return padding(convert_pad(update_pad_lower).sizes(),
-                   convert_pad(update_pad_upper).sizes(),
-                   0.0f,
-                   convert_pad(update_pad_mask));
+    // TODO: rework this method
```

Comment:
Please add more details if you have any improvement idea

Reply:
This function could be updated accordingly to padding class change. However I still need more time to understand it. Some question to its existing implementing, like - https://github.com/openvinotoolkit/openvino/pull/23690/files#r1728206375.